lab11_Mvc/Models/PersonOfTheYear.cs

```diff
@@ -16,5 +19,35 @@ public class PersonOfTheYear
 		public string Title { get; set; }
 		public string Category { get; set; }
 		public string Context { get; set; }
+
+
+		public static List<PersonOfTheYear> GetStaff(int selectedStartYear, int selectedEndYear)
+		{
+			string[] PersonOfTheYear = File.ReadAllLines("App_Data/personOfTheYear.csv");
+
+			return PersonOfTheYear
+			   .Skip(1) // skip the header
+			   .From(year in PersonOfTheYear)
+			   .where(selectedStartYear<= year && selectedEndYear>=year)
```

Comment:
1. You can't filter until you have a `PersonOfTheYear`; this needs to be right above the `ToList()`, after we have created our `PersonOfTheYear` objects.
2. `From()` doesn't actually exist. When we do this:

    ```csharp
    from person in PersonOfTheYear
    where selectedStartYear <= person.Year && person.Year <= selectedEndYear
    select person
    ```
    In method syntax, the `from` name is the name of the lambda parameter:

   ```csharp
   PersonOfTheYear
      // Select(...)
      .Where(person => selectedStartYear <= person.Year && person.Year <= selectedEndYear)
      .ToList();
   ```